### PR TITLE
docs(services): add `indexerUrl` to the indexer doc

### PR DESCRIPTION
**Pull request by qbzzt**

### CODE REVIEW

**Review by holic — APPROVED** [review]

File: docs/pages/services/indexer.mdx [hunk 1]
```diff
@@ -186,3 +186,10 @@ DROP OWNED BY <see below>;
 
 - If you are running the indexer locally, the owner of the tablespaces is your user name.
 - If you are using Docker, the owner of the tablespaces is `root`.
+
+## Using the indexer
+
+The source code of a MUD client has a call either to [`syncToRecs`](https://github.com/latticexyz/mud/blob/main/packages/store-sync/src/recs/syncToRecs.ts#L21-L82) or to [`syncToZustand`](https://github.com/latticexyz/mud/blob/main/packages/store-sync/src/zustand/syncToZustand.ts#L33-L74), typically in [`setupNetwork.ts`](https://github.com/latticexyz/mud/blob/main/templates/react/packages/client/src/mud/setupNetwork.ts#L74-L79).
+This call initializes the synchronization between the data source (blockchain or indexer) and the client's copy.
+
+To use the indexer, specify an `indexerUrl` parameter with the URL.
```
Comment (holic) [inline docs/pages/services/indexer.mdx:195]: do we want to provide an exampl ehere?

File: docs/pages/services/indexer.mdx [hunk 2]
```diff
@@ -186,3 +186,10 @@ DROP OWNED BY <see below>;
 
 - If you are running the indexer locally, the owner of the tablespaces is your user name.
 - If you are using Docker, the owner of the tablespaces is `root`.
+
+## Using the indexer
+
+The source code of a MUD client has a call either to [`syncToRecs`](https://github.com/latticexyz/mud/blob/main/packages/store-sync/src/recs/syncToRecs.ts#L21-L82) or to [`syncToZustand`](https://github.com/latticexyz/mud/blob/main/packages/store-sync/src/zustand/syncToZustand.ts#L33-L74), typically in [`setupNetwork.ts`](https://github.com/latticexyz/mud/blob/main/templates/react/packages/client/src/mud/setupNetwork.ts#L74-L79).
+This call initializes the synchronization between the data source (blockchain or indexer) and the client's copy.
```
Comment (holic) [inline docs/pages/services/indexer.mdx:193]:
```suggestion
This call initializes the synchronization between the data source (RPC or indexer) and the client's copy.
```

File: docs/pages/services/indexer.mdx (hunk 1 above)
Comment (qbzzt) [reply docs/pages/services/indexer.mdx:197]: Yes. Consider it done.

File: docs/pages/services/indexer.mdx [hunk 3]
```diff
@@ -211,3 +213,25 @@ DROP OWNED BY <see below>;
...
+<CollapseCode>
+
+```ts {7}
+const { components, latestBlock$, storedBlockLogs$, waitForTransaction } = await syncToRecs({
+  world,
+  config: mudConfig,
+  address: networkConfig.worldAddress as Hex,
+  publicClient,
+  startBlock: BigInt(networkConfig.initialBlockNumber),
+  indexerUrl: "http://127.0.0.1:3001/trpc",
```
Comment (alvrs) [inline docs/pages/services/indexer.mdx:233]:
```suggestion
  indexerUrl: "http://127.0.0.1:3001",
```

Comment (alvrs) [reply docs/pages/services/indexer.mdx:233]: (the `/trpc` it's not needed anymore with the latest MUD packages)

**Review by alvrs — DISMISSED** [review]

**Review by alvrs — APPROVED** [review]

**Pull request merged**